src/views/vwWriter/edit.ejs

```diff
@@ -50,43 +50,102 @@
+    <label for="status">Status</label>
+    <input type="text" 
+           id="status" 
+           class="form-control" 
+           disabled 
+           value="<%= article?.status || 'Draft' %>" 
+           name="status" />
+  </div>
+
+  <div class="form-group">
+    <label for="txtTitle">Title</label>
+    <input type="text" 
+           id="txtTitle" 
+           class="form-control" 
+           name="title" 
+           placeholder="Enter the title"
+           value="<%= article?.title || '' %>" />
+  </div>
+
+  <div class="form-group">
+    <label for="txtSummary">Summary</label>
+    <textarea id="txtSummary" 
+              class="form-control" 
+              name="summary" 
+              rows="3" 
+              placeholder="Write a brief summary"><%= article?.abstract || '' %></textarea>
+  </div>
+
+  <div class="form-group">
+    <label for="txtContent">Content</label>
+    <textarea id="txtContent" 
+              class="form-control" 
+              name="content" 
+              rows="5" 
+              placeholder="Write the content"><%= article?.content || '' %></textarea>
+  </div>
+
+  <div class="form-group">
+    <label for="thumbnail">Select a thumbnail</label>
+    <input type="file" 
+           class="form-control" 
+           name="thumbnail" 
+           id="thumbnail" 
+           accept="image/*" />
+    
+    <% if (article && article.thumbnail) { %>
+      <p class="mt-2">Current Thumbnail:</p>
+      <img src="/img/<%= article.thumbnail %>" 
+           alt="Current Thumbnail" 
+           class="mt-2" 
+           style="max-width: 200px;">
+      <p class="text-muted">Upload a new image to replace the current thumbnail if you want</p>
+    <% } else { %>
+      <p class="text-muted">No thumbnail uploaded yet. You can add one now.</p>
+    <% } %>
+  </div>
+  
+
+  <label for="category">Select category</label>
+  <select name="category" id="category" class="form-control">
+    <option value="category-1" class="font-weight-bold" 
+            <%= article?.category_id === 'category-1' ? 'selected' : '' %>>Category 1</option>
+    <option value="category-1-1" 
+            <%= article?.category_id === 'category-1-1' ? 'selected' : '' %>>&nbsp;&nbsp;Category 1-1</option>
+    <option value="category-1-2"
+            <%= article?.category_id === 'category-1-2' ? 'selected' : '' %>>&nbsp;&nbsp;Category 1-2</option>
+    <option value="category-2" class="font-weight-bold"
+            <%= article?.category_id === 'category-2' ? 'selected' : '' %>>Category 2</option>
+    <option value="category-2-1"
+            <%= article?.category_id === 'category-2-1' ? 'selected' : '' %>>&nbsp;&nbsp;Category 2-1</option>
+  </select>
+
+  <a href="/writer" role="button" class="btn btn-outline-danger mt-3">Cancel</a>
+  <button type="submit" class="btn btn-outline-success mt-3">
+    <%= article ? 'Update' : 'Create' %>
+  </button>
 </form>
+
+<script>
+  // Initialize TinyMCE
+  tinymce.init({
```

Comment:
tinymce is already been initialized on line 44, move the setup option onto the initialized tinymce.